CONTRIBUTING.md

```diff
@@ -266,6 +266,44 @@ tokio-console http://127.0.0.1:5555
 
 </details>
 
+### Profiling
+
+<details> <summary> Expand to learn ho to profile iroha. </summary> </details>
+
+To optimize performance it's useful to profile iroha.
+
+To do that you should compile iroha with `profiling` profile and with `wasm_profiling` compilation flag: 
+
+```bash
+RUSTFLAGS="-C force-frame-pointers=on --cfg wasm_profiling"cargo build --profile profiling
```

Comment:
It would be very nice to specify the `force-frame-pointers` in the cargo configuration, but, unfortunately, this is not stable yet =( https://github.com/rust-lang/cargo/issues/10271

Reply:
yes, this is what @Erigara did at first, but we have to keep iroha to build on `stable` so he removed it